tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1992,3 +1992,39 @@ def test_autolog_registering_model():
 
         registered_model = MlflowClient().get_registered_model(registered_model_name)
         assert registered_model.name == registered_model_name
+
+
+def test_autolog_pos_label_used_for_training_metric():
+    mlflow.sklearn.autolog(pos_label=1)
+
+    import sklearn.ensemble
+
+    model = sklearn.ensemble.RandomForestClassifier(max_depth=2, random_state=0, n_estimators=10)
+    X, y = sklearn.datasets.load_breast_cancer(return_X_y=True)
+
+    mlflow.start_run()
+    model = fit_model(model, X, y, "fit")
+
+    run_id = mlflow.active_run().info.run_id
+    _, training_metrics, _, _ = get_run_data(run_id)
+    expected_training_metrics = mlflow.sklearn.eval_and_log_metrics(
+        model=model, X=X, y_true=y, prefix="training_", pos_label=1
+    )
+    assert training_metrics == expected_training_metrics
+
+    mlflow.end_run()
+
+
+def test_autolog_emits_warning_message_when_pos_label_used_for_multilabel():
+    mlflow.sklearn.autolog(pos_label=1)
+
+    model = sklearn.svm.SVC()
+    X, y = get_iris()
+
+    with mlflow.start_run(), mock.patch("mlflow.sklearn.utils._logger.warning") as mock_warning:
+        model.fit(X, y)
+        mock_warning.called_once_with(
```

Comment:
```suggestion
        mock_warning.assert_called_once_with(
```

Reply:
There're actually 3 warnings, each for precision, recall and f1. I changed the test accordingly. 
I also found `called_once_with` actually doesn't do any assert, while it's used in several other tests, such as `test_autolog_max_tuning_runs_logs_info_correctly`. I'm not able to quickly fix the other tests to use `assert_called_once_with` in this PR, but we'll probably want to fix it some time.